components/inputs/test/input-number.perf.js

```diff
@@ -0,0 +1,37 @@
+import '../input-number.js';
+import '../../button/button-icon.js';
+import { html, testRenderTime } from 'web-test-runner-performance/browser.js';
+import { expect } from '@open-wc/testing';
+
+describe('d2l-input-number', () => {
+
+	it('normal', async() => {
+		const element = html`<d2l-input-number label="label" value="18"></d2l-input-number>`;
+		const result = await testRenderTime(element, { iterations: 500 });
+		expect(result.duration).to.be.below(1500);
+	});
+
+	it('min-max', async() => {
+		const element = html`<d2l-input-number label="label" value="18" min="18" max="100"></d2l-input-number>`;
+		const result = await testRenderTime(element, { iterations: 500 });
+		expect(result.duration).to.be.below(1500);
+	});
+	it('min-max-exclusive', async() => {
+		const element = html`<d2l-input-number label="label" value="5" min="5" max="100" min-exclusive max-exclusive></d2l-input-number>`;
+		const result = await testRenderTime(element, { iterations: 500 });
```

Comment:
This test was particularly slow so I increased the timeout to 2500. In this file I actually reduced the amount of iterations to 500 to speed things up. Any thoughts on if we want to keep all iteration values consistant and only change timeouts or some other option?

Reply:
And it looks like it's still failing at 2500, what a slow thing! I wonder if you've uncovered an actual performance problem?